recipes/kcodecs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "kcodecs" %}
+{% set version = "5.64.0" %}
+{% set major_minor = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://download.kde.org/stable/frameworks/{{ major_minor }}/kcodecs-{{ version }}.tar.xz
+  sha256: 24cbffb123179cf4386500ae7bc7c99f65c4422cd7b91f314152f11cd596402a
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('kcodecs', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - extra-cmake-modules
```

Comment:
Per previous comments I guess this goes in build.

Reply:
Probably good to adjust this everywhere preemptively.

Reply:
Yeah, I'll fix them all up!

Reply:
OK they are all updated!